sql/2019/10_SEO/10_10.sql

```diff
@@ -0,0 +1,21 @@
+#standardSQL
+
+# Linking - extract <a href> count per page (internal + external)
+
+CREATE TEMPORARY FUNCTION parseAnchor(payload STRING, element STRING)
```

Comment:
Similar to 10a, I would do this using standard `JSON_EXTRACT` functions since the values are simple ints.